extensions/cricketcast/src/index.jsx

```diff
@@ -0,0 +1,74 @@
+import { ActionPanel, OpenInBrowserAction, List, showToast, ToastStyle, CopyToClipboardAction } from "@raycast/api";
+import { useEffect, useState } from "react";
+import convert from "xml-js";
+import fetch from "node-fetch";
+
+function Actions({ item }) {
+  return (
+    <ActionPanel title={item.title}>
+      <ActionPanel.Section>{item.link && <OpenInBrowserAction url={item.link} />}</ActionPanel.Section>
+      <ActionPanel.Section>
+        {item.link && (
+          <CopyToClipboardAction content={item.link} title="Copy Link" shortcut={{ modifiers: ["cmd"], key: "." }} />
+        )}
+      </ActionPanel.Section>
+    </ActionPanel>
+  );
+}
+
+export default function Command() {
+  const [state, setState] = useState({});
+
+  useEffect(() => {
+    async function fetchStories() {
+      try {
+        let XMLdata = await fetch("http://static.cricinfo.com/rss/livescores.xml").then((r) => r.text());
+        let JSONdata = JSON.parse(await convert.xml2json(XMLdata, { compact: true, spaces: 4 }));
+        let matches = JSONdata.rss.channel.item.map((match) => ({
+          title: match.title["_text"].replace(" *", "*"),
+          id: match.guid["_text"].replace("http://www.cricinfo.com/ci/engine/match/", "").replace(".html", ""),
```

Comment:
```suggestion
          id: match.guid["_text"].replace( /\D+/g, ''),
```
Could be replaced with this instead, the current format is always the same.